packages/venia-concept/src/components/MiniCart/kebab.js

```diff
@@ -40,7 +40,7 @@ class Kebab extends Component {
         return (
             <div {...restProps} className={classes.root}>
                 <button className={classes.kebab} ref={this.kebabButtonRef}>
-                    <Icon src={MoreVerticalIcon} attrs={{ color: 'black' }} />
```

Comment:
So `black` isn't actually one of the colors in our palette. We don't use it for text, buttons, etc. The custom property `--venia-text` points to a dark gray that's nearly black, but isn't `#000` / `black`. In any case, color should applied via CSS.